vademecum.tex

```diff
@@ -325,17 +325,13 @@ \section{Masters in HPC and SC}
 
-\section{Temporary unemployment benefit}
+\section{Temporary unemployment benefit (DIS-COLL) }
 
-If you do not have a Gestiona Separata completed -- it is the first step to do.
-\begin{itemize}
-   \item The first way is to go directly to INPS office Trieste ( Via Sant'Anastasio, 5,), nearby busstop 38. Probably I didn't even fix an appointment- just came and had a talk. In principle, they should give you INPS code. But I had some problems, so I have obtained a SPID code (see (2.) )
-   \item One can obtain a SPID code online having the Tessera Sanitaria (your health card). If it is expired, it's also fine, the main thing is its' number. Although you should fill in the date of expiration of the health card of this year (like, September 2022 or December 2022), even it has already expired -- they do not check it further, the thing which is needed is a number.
-   \item Once you got your SPID, you go to the INPS website (https://www.inps.it/) and complete the procedure "Gestiona Separata".
-   \item Then, you go to Anpal website (https://www.anpal.gov.it/did) and complete the procedure "Did online"-- at the end they will ask you to send an email to schedule an appointment to the adress (cpi.trieste@regione.fvg.it). They will reply to you with a date of the appointment . Then you will go to the address (Scala dei Cappuccini, 1) and get a bunch of documents which one can also use to get a job seeking permit of stay.
-   \item Finally, you should make a diss-coll (https://www.inps.it/prestazioni-servizi/dis-coll-indennita-mensile-di-disoccupazione) procedure, where you will fill in with your IBAN , which is for getting the unemployment benefit.
-    As I have understood, one can do 4. and 5. at the same time, more or less.
-\end{itemize}
+Only who have Gestione Separata can ask for this benefit, that consists in a monthly allowance of the 75\% of the average month income of the recipient, up to a big enough maximum. If you do the request within 8 days from the end of the PhD contract (that can be, and usually is, before the PhD discussion!), INPS will give start to the benefit 8 days after the end of the contract. Otherwise the benefit is calculated from the day after the application is completed. The maximum duration of DIS-COLL is 12 month.
+
+
+If you do not have Gestiona Separata, this is the first step to do. You must have it if you are a PhD student (italian law requirement).
```

Comment:
If you are not subscribed to the INPS Gestione Separata, this is the first essential step to make. You must have it if you are a PhD studente, according to the Italian law requirements.